src/telemetry/__init__.py

```
@@ -0,0 +1,87 @@
+__all__ = [
+    # OTel metrics
+    "get_meter",
+    "initialize_otel_metrics",
+    "shutdown_otel_metrics",
+    "otel_metrics",
+    # CloudEvents
+    "emit",
+    # Lifecycle
+    "initialize_telemetry",
+    "shutdown_telemetry",
+]
```

Ruff (0.14.13)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)